package.json

```diff
@@ -59,7 +59,8 @@
     "json-loader": "0.5.4",
     "just-wait": "1.0.3",
     "webpack": "1.12.11",
-    "webpack-dev-server": "1.14.1"
+    "webpack-dev-server": "1.14.1",
+    "webpack-node-externals": "^0.4.1"
```

Comment:
Could you please remove the caret? 

(Whether the caret is necessary/preferred is out-of-scope of this PR. I'd like to merge it without caret first.)